### PR TITLE
Update channel.html

### PR DESCRIPTION
**Pull request by ahimani**
This PR fixes some grammar and spelling errors and restructures certain sentences to improve readability of the article

### CODE REVIEW

File: articles/channel.html [hunk 1]
```diff
@@ -488,7 +488,7 @@ <h3>Detailed Explanations For Channel Operations</h3>
 
 <u><b>Channel operation case A</b>:
 when a goroutine <code>Gr</code> tries to receive a value from a not-closed non-nil channel</u>,
-the goroutine <code>Gr</code> will acquire the lock accociated with the channel firstly,
+the goroutine <code>Gr</code> will acquire the lock associated with the channel firstly,
```
Comment (go101) [inline articles/channel.html]: Aha, there is another such typo at line 535. ;D

File: articles/channel.html [hunk 2]
```diff
@@ -301,7 +301,7 @@ <h3>Channel Operations</h3>
 
 	<p>
 	</p>
-	Two channel receive operations which are used as source values in assignments:
+	Two channels receive operations which are used as source values in assignments:
```
Comment (go101) [inline articles/channel.html]: Ah, I think here the "Two" is a qualifier of "operations". ;D

File: articles/channel.html (hunk 1 above)
Comment (ahimani) [reply articles/channel.html]: Done :)

File: articles/channel.html (hunk 2 above)
Comment (ahimani) [reply articles/channel.html]: Ah, that makes sense. Reverted the change.

**Pull request merged**